Client/Frontend/Login Management/BreachAlertsClient.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  BreachAlertsClient.swift
+//  Client
+//
+//  Created by Vanna Phong on 5/27/20.
+//  Copyright © 2020 Mozilla. All rights reserved.
+//
+
+import Foundation
+import Shared
+
+/// Errors related to BreachAlertsClient and BreachAlertsManager.
+struct BreachAlertsError: MaybeErrorType {
+    public let description: String
+    public init(description: String) {
+        self.description = description
+    }
+}
```

Comment:
Is this init required for the `MaybeErrorType` protocol conformance? If not, I don't believe you need it as structs in Swift come with initializers baked in for their variables.

Reply:
I was following LoginRecordError for this https://github.com/mozilla-mobile/firefox-ios/blob/4436268ba1c5fb3beea390e0b9b72b2c88a95fc8/Storage/Rust/RustLogins.swift#L78-L83
but now that I look around the codebase, there doesn't seem to be a standard for defining `MaybeErrorType`s. I took out the init and it compiles fine so I'll add this in the next commit